docs/editors/overview.md

```diff
@@ -478,15 +478,15 @@ scripts.
   </tr>
   <tr>
     <td>Find implementations</td>
-    <td align="center"></td>
+    <td align="center">✅</td>
     <td align="center"></td>
```

Comment:
`implementations` are not yet supported - added an issue here: https://github.com/scalameta/metals-feature-requests/issues/190